src/utils/scroll-offset.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * This source file is part of the Swift.org open source project
+ *
+ * Copyright (c) 2021 Apple Inc. and the Swift project authors
+ * Licensed under Apache License v2.0 with Runtime Library Exception
+ *
+ * See https://swift.org/LICENSE.txt for license information
+ * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import { documentationTopicName } from 'docc-render/constants/router';
+
+export const EXTRA_DOCUMENTATION_OFFSET = 10;
+
+/**
+ * @param {import('vue-router').Route} to
+ * @returns {Number}
+ */
+export default function getExtraScrollOffset(to) {
```

Comment:
Would it make sense to move this to the existing `router-utils.js` file where we already have utilities related to scrolling (and also offsets for scrolling) as opposed to creating an entirely new `scroll-offset.js` file?

Reply:
Thought it might be easier for future 3rd party users to extend this function 🤔